src/board/internal.tsx

```diff
@@ -102,14 +102,29 @@ export function InternalBoard<D>({
       dispatch({ type: "submit" });
 
       const removedItemIndex = items.findIndex((it) => it.id === removeTransition.removedItem.id);
-      const nextIndexToFocus = removedItemIndex !== items.length - 1 ? removedItemIndex : items.length - 2;
-      focusNextRenderIndexRef.current = nextIndexToFocus;
+      let nextIndexToFocus = -1;
```

Comment:
could be improved, having an initial logic to test the workflows and will add tests and a new page